docs/App.js

```diff
@@ -1,137 +1,59 @@
-      duration: 5000,
-      transitionDuration: 500,
-      indicators: true,
-      scale: 1.4,
-      pauseOnHover: true
-    };
-    const { slideImages, zoomOutImages, fadeImages } = this.state;
-    return (
-      <div>
-        <h3>Slide Effect</h3>
-        <div className="slide-container">
-          <Slide {...properties}>
-            {slideImages.map((each, index) => (
-              <div key={index} className="each-slide">
-                <div style={{ backgroundImage: `url(${each})` }}>
-                  <span>Slide {index + 1}</span>
-                </div>
-              </div>
-            ))}
-          </Slide>
-        </div>
-        <br />
-
-        <h3>Fade Effect</h3>
-        <div className="slide-container">
-          <Fade {...fadeProperties}>
-            <div className="each-fade">
-              <div className="image-container">
-                <img src={fadeImages[0]} />
-              </div>
-              <h2>First Slide</h2>
-            </div>
-            <div className="each-fade">
-              <h2>Second Slide</h2>
-              <div className="image-container">
-                <img src={fadeImages[1]} />
-              </div>
-            </div>
-            <div className="each-fade">
-              <div className="image-container">
-                <img src={fadeImages[2]} />
-              </div>
-              <h2>Third Slide</h2>
-            </div>
-          </Fade>
-        </div>
-
-        <br />
-        <h3>Zoom out Effect</h3>
-        <div className="slide-container">
-          <Zoom {...zoomOutProperties}>
-            {zoomOutImages.map((each, index) => (
-              <img key={index} style={{ width: '100%' }} src={each} />
-            ))}
-          </Zoom>
-        </div>
-        <br />
-        <h3>Zoom in Effect</h3>
-        <div className="slide-container">
-          <Zoom {...zoomInProperties}>
-            {fadeImages.map((each, index) => (
-              <img key={index} style={{ width: '100%' }} src={each} />
-            ))}
-          </Zoom>
-        </div>
-        <br />
+import React, { useState } from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import Navbar from "./views/components/NavBar";
+import Sidebar from "./views/components/SideBar";
+import ExamplesPage from "./views/pages/ExamplesPage";
+import LandingPage from "./views/pages/LandingPage";
+import ApiPage from "./views/pages/ApiPage";
+import ForTypescript from "./views/pages/ForTypescript";
+import SlideExample from "./views/pages//Examples/Slide";
+import FadeExample from "./views/pages//Examples/Fade";
+import ZoomExample from "./views/pages//Examples/Zoom";
+import Next from "./views/pages/Next";
+import "./views/styles.css";
+import "../src/css/styles.css";
+import "./assets/fonts/proxima-nova-regular.otf";
+
+const App = () => {
+  const [sidebar, setSidebar] = useState(false);
+  const sideBarItem = document.querySelector(".sidebar");
```

Comment:
Instead of using document.querySelector you can use `ref` with the `useRef` hook. That way we can leverage on React functionalities

Reply:
Actually, on second thought, I don't think we need to manipulate the margin value this way. You have a state that keeps track of the status (open or closed) of the sidebar. What we just need to do is conditionally apply the marginLeft style in the sidebar component.
For the window resize, when the screen is greater than 768 you just update the state to true and vice versa

Reply:
Got it.